index.html

```diff
@@ -0,0 +1,225 @@
+          with people from over 80 countries is taking place
+          in October, in Korea.
+        </p>
+        <h2 class="date">
+          2023.10.15(THU) ~ 16(FRI)
+        </h2>
+        <h3 class="address">
+          @  National Museum of Korea, Art Center Nabi and more .
+        </h3>
+      </div>
+    </section>
+    <!-- main header end  -->
+    <!-- main section start  -->
+    <section class="main_section" id="program_section">
+
+      <div class="main_section_div">
+        <h3 class="main_header_text">Main Program</h3>
+        <div class="line"></div>
+
+        <Div class="main_cards_div">
+
+          <div class="main_card">
+            <img class="main_card_icon" src="images/icons/icon_lecture.svg" alt="Lecture">
+              <h3 class="main_card_title">Lecture</h3>
+            <p class="main_card_message">
+              listen to the speakers from various countries
+              about the messeges of sharing and opening.
+            </p>
+          </div>
+
+          <div class="main_card">
+            <img class="main_card_icon" src="images/icons/icon_Exhibition.svg" alt="Exhibition">
+            <h3 class="main_card_title"> CC Exhibition</h3>
+            <p class="main_card_message">
+              Appreciate various creations applying CC
+              license of artists, organized from Art Center Nabi.
+            </p>
+          </div>
+
+          <div class="main_card">
+            <img class="main_card_icon" src="images/icons/icon_forum.svg" alt="forum">
+            <h3 class="main_card_title">Forum</h3>
+            <p class="main_card_message">
+              Have the time to share your thoughts and opinions with experts for each topic more.
+            </p>
+          </div>
+
+          <div class="main_card">
+            <img class="main_card_icon" src="images/icons/icon_workshop.svg" alt="workshop">
+            <h3 class="main_card_title">Workshop</h3>
+            <p class="main_card_message">
+              try creating your own work using open source license rather than just watching at it.
+            </p>
+          </div>
+
+          <div class="main_card">
+            <img class="main_card_icon" src="images/icons/icon_ignite.svg" alt="Ignite">
+            <h3 class="main_card_title">Ignite</h3>
+            <p class="main_card_message">
+              Get opportunities to network with CC
+              affiliates around the world, also after the summit
+            </p>
+          </div>
+        </div>
+
+        <button class="join_main_button" type="button">
+          Join CC Global Summit  2023
+        </button>
+
+        <a rel="" href="#" class="link_to_join"> <h4> SEE THE WHOLE PROGRAM </h4></a>
+
+      </div>
+    </section>
+      <!-- End of main section  -->
+      <!-- feature speaker start  -->
+    <section class="featured_speakers_section" id="speakers_section">
+      <h3 class="featured_speakers_header_text">
+        Feature speakers
+      </h3>
```

Comment:
- [ ] Great job on adding the future speakers with JavaScript! 👏 I just have a small suggestion to i**mprove the alignment of your project.** Could you please consider reducing the amount of text on your future speakers cards? I noticed that it doesn't look aligned on mobile devices. 😊

   ### Your design: 
   ![image](https://github.com/Nelofarzabi/Capstone_Project/assets/107952724/65133548-35a0-4bc3-b618-22e9f0407e2d)

   ### Behance design:
   ![image](https://github.com/Nelofarzabi/Capstone_Project/assets/107952724/45c447e9-111a-4f09-8ccf-2738a6ca5f4b)